frontend/src/theme/canada.js

```diff
@@ -100,6 +100,9 @@ export default extendTheme({
       800: '#153E75',
       900: '#0D467D',
     },
+    red: {
+      500: '#e50003',
+    },
```

Comment:
We can do a similar thing with red.500 as with the teals. To override FormErrorMessage, use:
```
const FormError = {
  parts: ['text', 'icon'],
  // Styles for the base style
  baseStyle: {
    text: {
      color: 'red.600',
    },
  },
  // Styles for the size variations
  sizes: {},
  // Styles for the visual style variations
  variants: {},
  // The default `size` or `variant` values
  defaultProps: {},
}

export default FormError
```